### PR TITLE
Ft mob desk version

### PR DESCRIPTION
**Pull request by freddykaberuka**
## Completed Task

- implementation of two pages(landing and about pages)
- implementation of mobile and Desktop version of those page
- add menu for mobile
- add dynamic data for speakers
- deploy an app with github page 
## Video Presentation
https://www.loom.com/share/2120eaa089e5407c81989e7f09b2869c

### CODE REVIEW

File: app.css [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+
+.btn {
+  color: #fff;
+  border: 1px solid #000;
+  padding: 30px;
+  font-weight: 700;
+  margin: 10px;
+  background-color: #ec5242;
+  font-size: 20px;
+}
```
Comment (CollinsTatang) [inline app.css]: - [x] Please remove all the big border on this button and make it very simple as the design present.
setfixbutton

File: app.css [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+
+.btn:hover {
+  background-color: #fff;
+  color: #ec5242;
+}
+
+/* SPEAKERS */
+#speakers {
+  margin: 10px;
+}
```
Comment (CollinsTatang) [inline app.css]: - [x] Kindly make the speaker's section center to the screen as seen in the design below.
center

### **Your Design**
spearkers

File: app.css [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+
+.copright {
+  padding: 0 20px;
+}
+
+.logos img {
+  width: 100%;
+  object-fit: cover;
+  position: relative;
+}
```
Comment (CollinsTatang) [inline app.css:307]: Please kindle restyle your footers to look like that of the design below. Remember to have the same footers structure cut across the different pages of your webside. The logo on the left and the text next to it. 
footersfix

### **Your Design**
footersCheck

File: about.html [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="about.css">
+    <title>CTF Summit</title>
+</head>
+<body>
+    <section id="landing-main">
+    <header>
+        <!-- <div id="hamburger"><i class="fa-solid fa-bars"></i></div> -->
```
Comment (CollinsTatang) [inline about.html]: - [x] Please remove all commented code in your codebase to make it clean and maintainable.

File: about.html [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="loggo">
+            <ul>
+                <li><img src="./assets/mozilla.png" alt="mozilla"></li>
+                <li><img src="./assets/google.png" alt="google"></li>
+                <li><img src="./assets/naver.png" alt="naver"></li>
+                <li><img src="./assets/daum.png" alt="daum"></li>
+                <li><img src="./assets/airbnb.png" alt="airbnb"></li>
+            </ul>
+            </div>
+        </section>
```
Comment (CollinsTatang) [inline about.html]: Please kindly follow the design structure and make your about page submit section look like the below design.
fixabout

### **Your Design**
submitCheck

File: index.html [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta2/css/all.min.css" integrity="sha512-YWzhKL2whUzgiheMoBFwW8CKV4qpHQAEuvilg9FAn5VJUDwKZZxkJNuGM4XkWuk94WCrrwslk8yWNGmY1EduTA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="app.css">
+    <title>CTF Summit</title>
+</head>
+<body>
+    <section id="landing-main">
```
Comment (CollinsTatang) [inline index.html]: - [x] Please remove this `section` tag and allow just the `header tag`. You don't wrap a header with a section for good practice of HTML semantics and high SEO performance.

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @freddykaberuka, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Suggested changes

- [ ] Kindly add a video presentation to your project describing the following below
projectVidoe


_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: app.css (hunk 2 above)
Comment (Emmyn5600) [reply app.css]: This is not well implemented, kindly refer to the Figma as well as the comment left by the previous reviewer

File: app.css (hunk 3 above)
Comment (Emmyn5600) [reply app.css:307]: Here, you can add some margin to display the text in the center of the image.

image

File: about.html (hunk 4 above)
Comment (Emmyn5600) [reply about.html]: Well done

File: index.html (hunk 6 above)
Comment (Emmyn5600) [reply index.html]: Here, seems like the logo has broken, kindly check what is causing this issue too.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @freddykaberuka 

Good job so far! I know you have worked hard on making this milestone. There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Required Changes :recycle:
- [ ] Check the comments under the review too.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 7]
```diff
@@ -1 +1,57 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```
Comment (gtekle) [inline README.md:53]: - [ Optional ] We are sorry for not pointing it earlier, but you should give credit to [Cindy Shin in Behance](https://www.behance.net/adagio07) for the original design idea as described in the project requirement. 
 
  > The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

File: app.js [hunk 8]
```diff
@@ -0,0 +1,110 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-unused-vars */
+/* eslint-disable linebreak-style */
```
Comment (gtekle) [inline app.js:3]: - [ OPTIONAL] You should not disable linters for they help you stick to best practice and have clean code base.

File: app.js [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+    id: 5,
+    p_name: 'Felix Kaberuka',
+    p_title: 'Chief Executive Officer of Kaspersky Lab',
+    p_history: 'He co-founded Kaspersky Lab in 1997 and helped identify instances of government-sponsored cyberwarfare',
+    p_image: './assets/eugene.jpg',
+  },
+];
+
+const speakerData = () => {
+  // eslint-disable-next-line array-callback-return
```
Comment (gtekle) [inline app.js:74]: - [ OPTIONAL] You should not disable linters so that you have a code based on best-practice and readable

**Review by gtekle — APPROVED** [review]
Hi @freddykaberuka :wave:, 

## Approved :clap: :clap: :clap: 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

clap-clapping

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**